packages/stream/create-stream/create-stream.test.js

```diff
@@ -0,0 +1,238 @@
+    it('should be a function', function () {
+      expect(createStream).to.be.a('function')
+    })
+  })
+
+  describe('Negative test', function () {
+    afterEach(function () {
+      return nock.cleanAll()
+    })
+
+    it('should return error if initialization instance & config not input', async function () {
+      try {
+        await createStream()
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to process because initialization is not valid and/or missing config. Please provide required initialization argument which is the initialization instance returned by the init() function and the configuration object'
+        )
+      }
+    })
+
+    it('should return error response if function is called with config argument in not object format', async function () {
+      const config = 'tes'
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to process because config argument must be in object format'
+        )
+      }
+    })
+
+    it('should return error response if function is called with no stream name argument', async function () {
+      const config = {}
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('Error')
+        expect(error.message).to.be.equal(
+          'Failed to create a new stream because the name of the stream is empty. Please provide a stream name'
+        )
+      }
+    })
+
+    it('should return error response if function is called with stream name with not a string type', async function () {
+      const config = { name: 2 }
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to create a new stream because the name of the stream is not in string format. A stream name must be in string format'
+        )
+      }
+    })
+
+    it('should return error response if function is called with stream slug with not a string type', async function () {
+      const config = { name: 'shopping', slug: 2 }
+      try {
+        await createStream(
+          init({
+            api_key: 'eyJhbGciOiJ.eyJleHAi.B01hriveOMR',
+          }),
+          config
+        )
+      } catch (error) {
```

Comment:
Same with the [comment on this PR](https://github.com/inlivedev/inlive-js-sdk/pull/12/files#r973824303). We shouldn't use `try catch` for testing